python/processing/algfactory.py

```diff
@@ -395,8 +395,8 @@ def dec(f):
 
     def output(self, type, *args, **kwargs):
         """
-        Define a output parameter for this algorithm using @alg.output.
-        Apart from `type` this method will take all arguments and pass them though to the correct `QgsProcessingOutputDefinition ` type.
+        Define an output parameter for this algorithm using @alg.output.
+        Apart from `type` this method will take all arguments and pass them though to the correct `QgsProcessingOutputDefinition` type.
```

Comment:
I couldn't decide if it should be "passed **through** (to)" or "passed ~~though~~ to"... Any help?